salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -21,6 +21,15 @@ Set up default basic auth htpasswd:
     - replace: False
 
 {%- set host = grains['metalk8s']['control_plane_ip'] %}
+{%- set etcd_servers = [] %}
+{%- for member in pillar.metalk8s.etcd.members | default([]) %}
```

Comment:
Please filterout etcd member without "name" (no name mean not started yet) (like in installed state of etcd)
`selectattr('name')`

Reply:
done